modin/pandas/test/test_groupby.py

```diff
@@ -127,8 +127,6 @@ def test_mixed_dtypes_groupby(as_index):
 
         # TODO Add more apply functions
         apply_functions = [lambda df: df.sum(), min]
-        # Workaround for Pandas bug #34656. Recreate groupby object for Pandas
```

Comment:
Does this removing relate to the issue?

Reply:
Yes. 
1) pandas bug https://github.com/pandas-dev/pandas/issues/34656 was closed. 
2) This workaround created additional issues related in case groupby is performed on index level (never tested before in this scope)